_projects/git_task_list.md

```diff
@@ -22,19 +22,19 @@ tags:
   - community
   - web
 ---
-The basic idea is an extended version of [GSOC Task List](http://github.com/summerofcode/gsoc-prep-tasks).
+The basic idea is an extended version of [GSOC Task List](https://github.com/coala/git-task-list)
```

Comment:
Seems you forgot a `.` there. 

And is there really a need to change this URL. They both are directing me to the same page.

Reply:
Nice catch!